ibis/expr/schema.py

```diff
@@ -143,6 +143,16 @@ def from_dict(cls, dictionary):
         names, types = zip(*dictionary.items()) if dictionary else ([], [])
         return Schema(names, types)
 
+    @classmethod
+    def merge(cls, schemas):
```

Comment:
Does this need to be a method on `Schema` as opposed to a utility function in your own codebase?

Reply:
It can be, of course. OTOH, I think it is useful and generic enough to be considered for inclusion in core. In any case, it's not a question we need to resolve for UDT design purposes.